assets/sass/components/global/_googlesitekit-cta-link.scss

```diff
@@ -26,9 +26,33 @@
 	text-align: left;
 	text-decoration: none;
 
+	svg {
+		fill: $c-link;
```

Comment:
Won't the `'currentColor'` in the SVG/here already make this work and use the colour specified below? In that case it might be a bit nicer/more concise/flexible since we won't need to duplicate the colour value for every style—and if we add more in the future it'll scale better. 🤔

Reply:
You are correct, thanks. I did overkill it with colour usage.